planter/planter.sh

```diff
@@ -44,7 +44,7 @@ set -o nounset
 # these can be overridden but otherwise default to the current stable image
 # used to build kubernetes from the master branch
 IMAGE_NAME="${IMAGE_NAME:-gcr.io/k8s-testimages/planter}"
-TAG="${TAG:-0.22.0}"
+TAG="${TAG:-0.23.0}"
```

Comment:
`docker pull gcr.io/k8s-testimages/planter:0.23.0` works

Reply:
Thank you for the review.